plugin/trino-accumulo/src/test/java/io/trino/plugin/accumulo/TestAccumuloConnectorTest.java

```diff
@@ -73,21 +79,21 @@ protected TestTable createTableWithDefaultColumns()
     }
 
     @Override
-    public void testCreateSchema()
+    public void testAddColumn()
     {
-        // schema creation is not supported
+        throw new SkipException("Adding columns via SQL are not supported until adding columns with comments are supported");
     }
 
     @Override
-    public void testAddColumn()
+    public void testDropColumn()
     {
-        // Adding columns via SQL are not supported until adding columns with comments are supported
+        throw new SkipException("Dropping columns are not supported by the connector");
     }
 
     @Override
-    public void testDropColumn()
+    public void testRenameTableAcrossSchema()
```

Comment:
Use `SUPPORTS_RENAME_TABLE_ACROSS_SCHEMAS` to control this test instead of override.

Reply:
On `TestAccumuloConnectorTest.java#66` I've already added `SUPPORTS_RENAME_TABLE_ACROSS_SCHEMAS` in `hasBehaviour()` method, but this is not enough to skip the test.

BaseConnectorTest.java
```
        if (!hasBehavior(SUPPORTS_RENAME_TABLE_ACROSS_SCHEMAS)) {
            if (!hasBehavior(SUPPORTS_RENAME_TABLE)) {
                throw new SkipException("Skipping since rename table is not supported at all");
            }
```

Reply:
If you have a look at my previous comment https://github.com/trinodb/trino/pull/7569#discussion_r612276312 you can see that Accumulo does support renaming a table.

Reply:
We don't need to skip the test entirely. If rename is supported then this should pass:

```
        if (!hasBehavior(SUPPORTS_RENAME_TABLE_ACROSS_SCHEMAS)) {
            if (!hasBehavior(SUPPORTS_RENAME_TABLE)) {
                throw new SkipException("Skipping since rename table is not supported at all");
            }
            assertQueryFails("ALTER TABLE nation RENAME TO other_schema.yyyy", "This connector does not support renaming tables across schemas");
            return;
        }
```

But I'm assuming it does not because the Accumulo connector doesn't explicitly prevent renames across the schema - this suggests the impl is incorrect and needs to be fixed - compare https://github.com/trinodb/trino/blob/7fd62db50aec490907e93412441fe65dbfea6052/plugin/trino-accumulo/src/main/java/io/trino/plugin/accumulo/AccumuloMetadata.java#L131-L141 vs https://github.com/trinodb/trino/blob/f19bf745ae8d0f1580807d3fba50011d9bcd2b49/plugin/trino-postgresql/src/main/java/io/trino/plugin/postgresql/PostgreSqlClient.java#L297-L309.

Reply:
@hashhar  this seems indeed to be a bug. I can handle it in a different issue. Should I / you create this new issue?

Reply:
IT's a one line change at https://github.com/trinodb/trino/blob/89e22cc33d21674d94372a0174743bf3d45a0b65/plugin/trino-accumulo/src/main/java/io/trino/plugin/accumulo/AccumuloClient.java#L475. The message needs to be changed to match the one in the test.

Reply:
Done.